neo.UnitTests/UT_MemoryPool.cs

```diff
@@ -65,11 +66,16 @@ private Transaction CreateHighPriorityTransaction()
 
         private Transaction CreateLowPriorityTransaction()
         {
-            long rNetFee = LongRandom(0, 100000, TestUtils.TestRandom);
+            long rNetFee = LongRandom(0, 10000, TestUtils.TestRandom);
             // [0,0.001] GAS a fee lower than the threshold of 0.001 GAS (not enough to be a high priority TX)
             return CreateTransactionWithFee(rNetFee);
         }
 
+        private bool IsLowPriority(Transaction tx)
+        {
+            return tx.FeePerByte < 1000;
```

Comment:
missed one 0? `10000` maybe

Reply:
These numbers are the same as they are now.